src/components/Shows/CheckBox/styles.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables.scss';
```

Comment:
No need to import `variables.scss` unless you use any CSS variables from the `variables.scss` file :-)